easybuild/scripts/bootstrap_eb.py

```diff
@@ -259,24 +298,34 @@ def stage1(tmpdir):
         os.environ['PYTHONPATH'] = os.pathsep.join([pkg_egg_dir] + pythonpaths)
 
         # determine per-package versions based on egg dirs
-        version_regex = re.compile('easybuild_%s-([0-9a-z.-]*)-py[0-9.]*.egg' % pkg)
+        version_regex = re.compile('%s-([0-9a-z.-]*)-py[0-9.]*.egg' % pkg.replace('-', '_'))
         pkg_egg_dirname = os.path.basename(pkg_egg_dir)
         res = version_regex.search(pkg_egg_dirname)
         if res is not None:
             pkg_version = res.group(1)
-            versions.update({'%s_version' % pkg: pkg_version})
             debug("Found version for easybuild-%s: %s" % (pkg, pkg_version))
+            if sources_path is None:
+                # downloaded tarballs should be versioned
+                templates.update({pkg: "'%s-%s.tar.gz'," % (pkg, pkg_version)})
+            else:
+                # names of specified source tarballs should not contain a version
+                templates.update({pkg: "'%s.tar.gz'," % pkg})
         else:
-            error("Failed to determine version for easybuild-%s package from %s with %s" % (pkg, pkg_egg_dirname, version_regex.pattern))
+            # vsc-base is only required for EasyBuild v2.x or higher
```

Comment:
not until it's released, and it should be possible to bootstrap older versions too (by supplying the tarballs for v1.16.1, for example)